bciers/apps/dashboard/middlewares/withAuthorizationDashboard.test.ts

```diff
@@ -185,17 +187,21 @@ describe("withAuthorizationDashboard middleware", () => {
     expect(result?.status).toBe(307);
   });
 
-  it("calls NextMiddleware for authenticated, NON-authorized cas_user if the route is /dashboard", async () => {
+  it("calls NextMiddleware for authenticated, NON-authorized cas_user if the route is in the allowed list", async () => {
```

Comment:
When this bug came up in standup, you said something about the tests not catching it. Is this the test you meant, and do these changes cover regression? It looks like the  dashboard page test (bciers/apps/dashboard/tests/routes/dashboard/page.test.tsx) doesn't test cas-pending

Reply:
@BCerki 

This is the test: `it("renders the dashboard-pending-message card for cas_pending role", async () => {` in file `bciers/apps/dashboard/tests/routes/dashboard/page.test.tsx`

I assumed it would fail if page `bciers/apps/dashboard/app/dashboard/page.tsx` fn ` data = await evalDashboardRules(data);` failed but, it does not...assumption wrong!

Reply:
Gotcha, it must be something about the mock. e2e is probably a better choice then, yes